### PR TITLE
t/porting/test_testlist.t - test that we test the same thing with t/harness and t/TEST

### PR DESCRIPTION
**Pull request by demerphq**
Also that we test everything expected in MANIFEST.

Also includes some fixups to t/TEST to deal with the fact that List/Util is not anymore the name of a distribution even though it is the name of an extension. Same for Cwd.

### CODE REVIEW

**Comment by demerphq** [comment]
@tonycoz, @xenu, @craigberry can you guys check this on your Win32 builds and on VMS please? I suspect this fixes 'make test' on win32. (Thanks to @xenu's feedback).

File: t/TEST [hunk 1]
```diff
@@ -119,35 +119,53 @@ $| = 1;
 #BEGIN { require '../lib/warnings.pm'; "warnings"->import() };
 
 # remove empty elements due to insertion of empty symbols via "''p1'" syntax
```
Comment (xenu) [inline t/TEST]: This comment should be moved to line 128.

**Review by xenu — CHANGES_REQUESTED** [review]

**Comment by tonycoz** [comment]
> 'make test' on win32

I see the hooks and class tests in CI, it looks fine except for the comment xenu mentioned.

File: t/TEST (hunk 1 above)
Comment (demerphq) [reply t/TEST]: Thanks, done.

**Comment by demerphq** [comment]
@xenu, i updated as requested. You ok with this now?

**Review by xenu — APPROVED** [review]

**Pull request merged**